pages/add/form/auto-complete-input.js

```diff
@@ -30,6 +31,24 @@ export default class AutoCompleteInput extends Component {
     };
   }
 
+  /**
```

Comment:
probably needs to be in this file instead: https://github.com/mozilla/network-pulse/blob/master/pages/add/form/tags.js#L4